website/docs/r/application_load_balancer_association.html.markdown

```diff
@@ -0,0 +1,94 @@
+---
+subcategory: "Service Networking"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_application_load_balancer_association"
+description: |-
+  Manages an Application Gateway for Containers Association.
+---
+
+# azurerm_application_load_balancer_association
+
+Manages an Application Gateway for Containers Association.
```

Comment:
since this is a _Subnet_ association, this resource probably should be `_subnet_association`?